Source/Core/AudioCommon/OpenALStream.h

```diff
@@ -78,14 +76,12 @@ class OpenALStream final : public SoundStream
   std::thread thread;
   Common::Flag m_run_thread;
 
-  Common::Event soundSyncEvent;
+  Common::Event sound_sync_event;
 
-  short realtimeBuffer[OAL_MAX_SAMPLES * STEREO_CHANNELS];
-  float sampleBuffer[OAL_MAX_SAMPLES * SURROUND_CHANNELS * OAL_MAX_BUFFERS];
-  ALuint uiBuffers[OAL_MAX_BUFFERS];
-  ALuint uiSource;
-  ALfloat fVolume;
+  std::vector<short> realtime_buffer;
```

Comment:
I added the "m_" prefix to all global but private variables. Should I also add it to all local variables?